src/data/clients.ts

```diff
@@ -124,13 +124,13 @@ export const clients: ClientData[] = [
   },
   {
     description:
-      "From Colorful Notion: Would love to connect our working QUIC implementation in Fall, organize a weeklong meetup at Devcon in Thailand.",
-    homepage: "https://dune.com/substrate/jam-duna",
+      "Colorful Notion's JAM.  TestNet collaborations welcome.  Currently closed source.",
```

Comment:
Removing doubled whitespace.

```suggestion
      "Colorful Notion's JAM. TestNet collaborations welcome.  Currently closed source.",
```